elasticsearch/examples/6.x/Makefile

```diff
@@ -13,3 +13,4 @@ test: install goss
 
 purge:
 	helm del --purge $(RELEASE)
+	for pvc in $$(kubectl get pvc -o name | grep $(RELEASE)); do kubectl delete $$pvc; done
```

Comment:
Grep is potentially dangerous since it will match partial release names. If someone has a release called `elasticsearch` it will delete all PVCs with `elasticsearch` in the name. Is it possible to do this based on labels like this?:

https://github.com/elastic/helm-charts/blob/340868c54c44ae97fd9ef8fa6d682472098dc816/helpers/examples.mk#L3-L7

Reply:
fixed in https://github.com/elastic/helm-charts/pull/336/commits/1e5616b3a8050b4810a2ff4974a01d22c207aab1